sci-biology/ensembl-compara/ensembl-compara-9999.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit git-r3 perl-functions
+
+DESCRIPTION="The Ensembl Compara Perl API and SQL schema"
+HOMEPAGE="http://www.ensembl.org/info/docs/api/"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/Ensembl/${PN}.git"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE=""
+
+DEPEND="sci-biology/bioperl"
+RDEPEND="${DEPEND}
+	sci-biology/ensembl"
+
+src_install() {
+	perl_set_version
+	insinto /usr/lib/perl5/${PERL_VERSION}
+	doins -r modules/*
+	rm -r "${ED}"/usr/lib/perl5/${PERL_VERSION}/t || die
```

Comment:
perhaps either exclude ```t``` from doins above or remove it before. Same for the other ebuilds in this PR.